src/types/models/BalancedGrowthCommunityModel.jl

```diff
@@ -0,0 +1,287 @@
+"""
+$(TYPEDEF)
+
+A standardized structure used to package models that can easily be combined into
+a [`BalancedGrowthCommunityModel`](@ref).
+
+# Fields
+$(TYPEDFIELDS)
+
+# Assumptions
+1. Exchange reactions, *all* of which are idenitified in `exchange_reaction_ids`,
+   have the form: `A[external] ⟷ ∅` where `A` is a metabolite. No other
+   exchanges are allowed.
+2. The biomass reaction of a model produces a biomass metabolite called
+   `biomass_metabolite_id` with stoichiometry 1.
+3. There is only one biomass reaction in the model.
+"""
+Base.@kwdef mutable struct CommunityMember
+    "Name of model appended to intracellular reactions and metabolites."
+    id::String
```

Comment:
TODO for myself, for later:
`id` should be outside of the community member, making the `members` a dict.